releasenotes/notes/compress-indentical-pulses-cd20e82e1525b9f8.yaml

```
@@ -0,0 +1,21 @@
+---
+features:
+  - |
+    Schedule transformer :func:`qiskit.pulse.transforms.compress_pulses`
+    performs an optimization pass to replace identical instances of
+    a pulse in a pulse schedule that improves the usage of waveform
+    memory in hardware and reduce confusion of pulse
+    users running into waveform limits on hardware.
```

Good, can be a little less awkward:
```suggestion
    Schedule transformer :func:`qiskit.pulse.transforms.compress_pulses`
    performs an optimization pass to reduce the usage of waveform
    memory in hardware by replacing multiple identical instances of
    a pulse in a pulse schedule with a single pulse.
```